ui/components/show/showCards/ShowCollection.tsx

```diff
@@ -0,0 +1,60 @@
+import Card from "../../layout/cardElements/card/Card";
+import CardCollection from "../../layout/cardElements/cardCollection/CardCollection";
+import CardContent from "./cardContent/CardContent";
+import { Show } from "../../../hooks/queries/shows";
+import { A11y, Navigation } from 'swiper/modules';
+import { Swiper, SwiperSlide } from 'swiper/react';
+import 'swiper/css';
+import 'swiper/css/a11y';
+import styles from './ShowCollection.module.scss';
+import cardCollectionStyles from '../../layout/cardElements/cardCollection/CardCollection.module.scss';
+import cardStyles from '../../layout/cardElements/card/Card.module.scss';
```

Comment:
CardContent was changed to ShowCollection. I'd still like to get rid of it all together by merging it into the less specific CardCollection in the future.

To make this component more generic, and avoid these non-co-located css module imports.

Should probably use composition to share card width styles between the Card component and SwiperSlide.